fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/rdf/impl/PropertiesRdfContextTest.java

```diff
@@ -0,0 +1,118 @@
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import javax.jcr.Session;
+
+import org.fcrepo.kernel.identifiers.IdentifierConverter;
+import org.fcrepo.kernel.impl.testutilities.TestPropertyIterator;
+import org.fcrepo.kernel.models.FedoraBinary;
+import org.fcrepo.kernel.models.FedoraResource;
+import org.fcrepo.kernel.models.NonRdfSourceDescription;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import com.hp.hpl.jena.rdf.model.Model;
+import com.hp.hpl.jena.rdf.model.Resource;
+/**
+ * <p>
+ * PropertiesRdfContextTest class.
+ * </p>
+ *
+ * @author yinlinchen
+ */
+public class PropertiesRdfContextTest {
+
+    @Mock
+    private FedoraResource mockResource;
+
+    @Mock
+    private FedoraBinary mockBinary;
+
+    @Mock
+    private Node mockNode;
+
+    @Mock
+    private Node mockBinaryNode;
+
+    @Mock
+    private NonRdfSourceDescription mockNonRdfSourceDescription;
+
+    @Mock
+    private Session mockSession;
+
+    private IdentifierConverter<Resource, FedoraResource> idTranslator;
+
+    private Resource mockContentSubject;
+
+    private Resource mockSubject;
+
+    @Before
+    public void setUp() throws RepositoryException {
+        initMocks(this);
+
+        when(mockResource.getNode()).thenReturn(mockNode);
+        when(mockNode.getSession()).thenReturn(mockSession);
+        when(mockNode.getProperties()).thenReturn(new TestPropertyIterator());
+
+        when(mockBinary.getNode()).thenReturn(mockBinaryNode);
+        when(mockBinaryNode.getSession()).thenReturn(mockSession);
+        when(mockBinary.getDescription()).thenReturn(mockNonRdfSourceDescription);
+        when(mockBinaryNode.getProperties()).thenReturn(new TestPropertyIterator());
+
+        idTranslator = new DefaultIdentifierTranslator(mockSession);
+        mockSubject = idTranslator.reverse().convert(mockResource);
+        mockContentSubject = idTranslator.reverse().convert(mockBinary);
+
+    }
+
+
+    /*
+     * (non-Javadoc) test the nonRdfSource response are included both NonRdfSourceDescription and RdfSourceDescription
+     */
+    @Test
+    public void testFedoraBinaryProperties() throws RepositoryException {
+        final Model results = new PropertiesRdfContext(mockBinary, idTranslator).asModel();
+
+        assertTrue("Response contains both NonRdfSourceDescription and RdfSourceDescription", results
+                .contains(mockSubject, DESCRIBES, mockContentSubject) && results
+                .contains(mockContentSubject, DESCRIBED_BY, mockSubject));
+
+    }
+
+    /*
+     * (non-Javadoc) test the FedoraResource response are included only RdfSourceDescription
+     */
+    @Test
+    public void testFedoraResourceProperties() throws RepositoryException {
+
+        final Model results = new PropertiesRdfContext(mockResource, idTranslator).asModel();
+        assertTrue("Response contains RdfSourceDescription", results
```

Comment:
Use the same two assertions from the test above, but make the second assertion be: `assertFalse(...)`